client/asset/eth/eth.go

```diff
@@ -107,6 +111,10 @@ func (d *Driver) Info() *asset.WalletInfo {
 	return WalletInfo
 }
 
+func (d *Driver) Create(params *asset.CreateWalletParams) error {
```

Comment:
Possible to move this to before `Open`? Also in other asset files please, to give visual representation of the order in which the methods are/should be called.